std/node/buffer_test.ts

```diff
@@ -0,0 +1,145 @@
+const { test } = Deno;
```

Comment:
```suggestion
```

https://github.com/denoland/deno/pull/6268 was just merged. It is best to use `Deno.test` instead of destructuring.

Reply:
Done.